apps/functions-subscription/api/internal.yaml

```diff
@@ -391,15 +415,57 @@ components:
           $ref: '#/components/schemas/TrialName'
         description:
           $ref: '#/components/schemas/TrialDescription'
+    TrialState:
+      type: string
+      description: |-
+        - CREATING: The Trial System is creating the resources related to the trial.
+          The configuration is not yet completed.
+        - CREATED: The Trial System created the resources for the trial.
+      enum:
+        - CREATING
+        - CREATED
+    CreatedTrial:
+      type: object
+      required:
+        - id
+        - state
+      properties:
+        id:
+          $ref: '#/components/schemas/TrialId'
+        state:
+          $ref: '#/components/schemas/TrialState'
     Trial:
       type: object
       required:
         - id
         - name
+        - state
       properties:
         id:
           $ref: '#/components/schemas/TrialId'
         name:
           $ref: '#/components/schemas/TrialName'
         description:
           $ref: '#/components/schemas/TrialDescription'
+        state:
+          $ref: '#/components/schemas/TrialState'
+        queue:
+          type: object
+          description: All the information the caller must have in order to properly
+            read events from the queue.
+            Theis object is available when the resources have been created, so when
+            the `state` property is `CREATED`
+          required:
+            - name
+            - identity
+          properties:
+            name:
+              type: string
+              description: The name of the queue where the events are sent.
+              minLength: 1
+            identity:
+              type: string
+              description: >-
+                The identifier of the Identity to use in order to read
+                the events from the queue.
```

Comment:
**thought**: maybe we can add this information with next PR.

It will be backward compatible since we are going to add stuff, not removing or renaming

Reply:
Yes, probably the best choice